src/cmd/api/controllers/videos_list_handler.go

```diff
@@ -106,14 +115,15 @@ func (v VideosListHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	order := vars["order"].(bool)
 	page := vars["page"].(int)
 	limit := vars["limit"].(int)
+	status := vars["status"].(models.VideoStatus)
 
 	log.Debug("GET VideosListHandler")
 
 	//Initialize the response
 	response := VideoListResponse{}
 
 	//Get videos to be returned
-	videos, err := v.VideosDAO.GetVideos(r.Context(), attribute, order, page, limit, int(models.COMPLETE))
+	videos, err := v.VideosDAO.GetVideos(r.Context(), attribute, order, page, limit, int(status))
```

Comment:
So user can now list video with any status?

Reply:
Yes, from the API. The front doesn't let you the option but we can easily add it. (in file: `/Voogle/src/webapp/src/components/PageNavigation.vue:30`)